tests/features/collection/collection_overview.feature

```diff
@@ -2,17 +2,13 @@
 Feature: Collections Overview
 
   Scenario: Check visibility of "Collections" menu link.
-    Given I am an anonymous user
-    Then I should see the link "Collections"
-    When I click "Collections"
-    Then I should see the heading "Collections"
-    # Check that all logged in users can see and access the link as well.
+    # Check that all logged in users can see and access the link.
```

Comment:
ISAICP-2639 is a ticket I created. You should link it in this file using a todo to explain why the anonymous user has been removed because we will need to implement this separately.!